content/en/docs/quickstart.md

```diff
@@ -29,19 +32,36 @@ docker build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks
 docker push localhost:5000/net-monitor:v1
 ```
 
+Tags are mutable and a tag reference can point to a different container image than the one signed. Find out the digest value from the output of `docker push` command.
```

Comment:
```suggestion
Save the digest value of the image from the output of the `docker push` command.

*Important*: Always use the digest value of an image when signing since they are immutable. Tag values are mutable and can reference a different container image than the original signed container image.
```